Makefile

```diff
@@ -940,3 +941,99 @@ define github_pr_add_comment
 	$(eval JSON := {"body":"$(3)"})
 	$(call github_call_api,POST,$(1)/issues/$(2)/comments,$(JSON))
 endef
+
+#####################################
+#####################################
+# ImageSet utility targets
+.PHONY: clean-imageset
+clean-imageset:
+	rm -f $(BUILD_DIR)/*imageset*
+
+
+ifdef VERSION
+OPERATOR_IMAGE ?= $(IMAGE_REGISTRY)/$(BUILD_IMAGE):$(VERSION)
+else
+OPERATOR_IMAGE ?= $(BUILD_IMAGE):latest
+endif
+CRANE_IMAGE ?= calico/go-build:master
```

Comment:
I think we should be using GO_BUILD_VER here. If that crane version is not right, then we should update it there too.

nit: could you add an empty line above to separate the OPERATOR_IMAGE and CRANE_IMAGE variable creations?

Reply:
We already have a way of defining the crane command in other repos:
https://github.com/projectcalico/calico/blob/53af4139e9df29001d36712ed900fb963261b064/lib.Makefile#L423-L441

It has the advantage of being tested and used already and not leaving extra files on your disk.

Reply:
What do you think should be copied from that because I don't think we should start including that lib.Makefile because that'll be introducing a dependency we don't need.
Do we need the "LOCAL" functionality? Do we need the "DRY_RUN"? I would probably suggest neither are needed since this is adding a convenience utility.

Reply:
We do already have this CONFIRM/DRY_RUN logic in our makefile, although I agree that we don't need it. Importing the other makefile could allow us to clean quite some thing up, and on the flip-side introduce things we don't need. Probably worth discussing separately at some point?

I think the thing we would need is `CRANE_CMD` and `double_quote`.